tests/integration/components/bs-button-test.js

```diff
@@ -76,6 +77,16 @@ module('Integration | Component | bs-button', function(hooks) {
     assert.dom('button i').hasClass('fa-check');
   });
 
+  testBS4('button with outline property shows as outline', async function(assert) {
+    await render(hbs`{{bs-button type="primary" outline=true}}`);
+    assert.dom('button').hasClass('btn-outline-primary');
+  });
+
+  testBS4('button with does not have outline class when outline is null', async function(assert) {
+    await render(hbs`{{bs-button type="primary"}}`);
+    assert.dom('button').doesNotHaveClass('btn-outline-primary');
```

Comment:
I don't think we need a separate test for this. If you want you can add this assertion to the existing `'button has correct type'` test.